### DISCUSSION TITLE
Can anyone launch Grand Theft Auto (GTA) V in Linux (wine)?

### DISCUSSION THREAD

**Discussion by TheDcoder** [legendary-gl/legendary#294] [Q&A] [answered]
Hi,

I downloaded all of GTA V yesterday and I am getting this error when I launch it:
![image](https://user-images.githubusercontent.com/6347576/122856661-cada5b00-d334-11eb-9bcf-ae09534095e6.png)

```
Unable to launch your game, please verify your game
data.
              [OK]    [Help]
```
According to my research, this is generated by Rockstar's Social Club application.

Has anyone been able to successfully get past this to a point where GTA itself would start? (would it work or crash is another issue)

**Comment by DuttaAB-dev** [comment]
Facing the same issue. Do you have winetricks installed? In my case this was usually solved by installing winetricks and relaunching.

**Reply by TheDcoder** [reply]
Yes, I also installed a bunch of stuff that was mentioned in Wine's AppDB for Epic Games Launcher but that did not help. Perhaps this needs the Proton version of wine or something?

**Comment by CommandMC** [comment, marked as answer]
I've written up what worked for me over here:
https://github.com/CommandMC/EpicLinux/wiki/GTA-V

**Reply by TheDcoder** [reply]
Thanks, did you get the same error dialog when you tried this with vanilla wine?

**Reply by CommandMC** [reply]
I can remember seeing that dialog more than once in the past, but it never appeared again after I followed those steps exactly. Even if you could get it to launch with wine-staging, Online play most likely won't work since it requires a special patch only Proton-GE has implemented.

**Reply by TheDcoder** [reply]
I tried Proton-GE as per your suggestion, the latest 6.10 version didn't work (like mentioned in your wiki) but fortunately the linked 6.9 version did!

Turns out Social Club didn't even install properly in vanilla Wine, it installed properly with all the stuff in a new prefix I created, but I am now stuck at the login screen and I can't enter any text in the login fields :angry:
![image](https://user-images.githubusercontent.com/6347576/123032447-94b7dc80-d403-11eb-991b-6aa52f43b01b.png)

I think their embedded Chrome browser is having issues with wine.

**Reply by TheDcoder** [reply]
I managed to fix the issue, I had to install some fonts and a winetricks packaged called `win10`:
```
$ winetricks arial cjkfonts win10
$ winetricks corefonts # only if the above didn't fix the issue
```
Credit to whoever wrote the [Lutris installation script](https://lutris.net/games/install/20468/view) :smile:

Also, I had to switch to vanilla wine when invoking `winetricks` and `setup_dxvk`, just mentioning in case someone tries to use the Proton wine for all of the stuff, because it won't work.

Edit: Forgot to mention the most important bit, the game is running fine once the launcher is fixed! :partying_face:

**Reply by CommandMC** [reply]
`win10` only sets your reported Windows version to Win 10 (which you can also do inside `winecfg`)
You can set the Wine version used with winetricks with the `WINE` environment variable (set it to the path of the actual executable). DXVK is a little more difficult, you'll have to add `PATH=/path/to/wine/version/folder:$PATH` before the `setup_dxvk` command.

I tend to avoid the Lutris install scripts since they add a lot of unnecessary stuff. In my case, none of those verbs were necessary to get the game running. I'll wait for more tests to come in before adding them to the Wiki page.

Good to hear you got it working!

**Reply by TheDcoder** [reply]
> `win10` only sets your reported Windows version

I expected it to do more than that, but okay, thanks for mentioning.

> You can set the Wine version used with winetricks with the WINE environment variable

That's cool, but I just modified my `PATH` for both commands.

> I tend to avoid the Lutris install scripts since they add a lot of unnecessary stuff. In my case, none of those verbs were necessary to get the game running.

Those fonts were definitely required in my case, because it only started working after I installed those. Perhaps you have MS Fonts already installed at system-level?
